docs/source/algorithms/CreateSampleShape-v1.rst

```diff
@@ -1,54 +0,0 @@
-.. algorithm::
```

Comment:
I assume you deleted this is you could build the docs, I don't think this should be committed though.